web/src/SidebarIcon.test.tsx

```diff
@@ -6,9 +6,12 @@ import { Color } from "./constants"
 
 type Ignore = boolean
 
-const cases: Array<
-  [string, ResourceStatus, Color | Ignore, IconType | Ignore]
-> = [
+const cases: Array<[
```

Comment:
In order to use the features I wanted from TypeScript 3.7 I had to upgrade prettier, which caused this formatting change.